app/test_main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import pytest
+
+from unittest import mock
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "valid_url,internet_connection,expected",
+    [
+        (True, True, "Accessible"),
+        (False, True, "Not accessible"),
+        (True, False, "Not accessible"),
+        (False, False, "Not accessible"),
+    ]
+)
+def test_can_access_google_page_if_internet_connection_and_valid_url(
```

Comment:
I think you better split it into several tests

Reply:
Hello!
I tried to think how to split this into different tests, but nothing came to mind.
To test if function `can_access_google_page` returns correct value, I need to mock both functions. If I don't test would not work properly.

Please, give some extra hints)

Reply:
you can use `id` in `parametrize` to describe your test